core/electron-manager/src/backend/ElectronHost.ts

```diff
@@ -114,7 +114,12 @@ export class ElectronHost {
     let assetPath = requestedUrl.substr(this._electronFrontend.length);
     if (assetPath.length === 0)
       assetPath = "index.html";
-    assetPath = assetPath.replace(/(#|\?).*$/, "");
+    let regexp = ["..", "#", "?", "*", "$"];

```

Comment:
I think there's probably a simpler way to do this...

Note that below, we use `fs.realpathSync` below to get a fully resolved _absolute_ path.  So we don't need to ban relative paths or other characters, we just need to check that the resolved path is still inside webResourcesPath, right?